packages/geometry/src/point.ts

```diff
@@ -89,6 +89,69 @@ export function scale(pt: Point, scaleX: number, scaleY: number): Point {
   };
 }
 
+/**
+ * Returns a new `Point` where `x` and `y` are multiplied by the given scale
+ * factor.
+ */
+export function scaleProportional(pt: Point, scale: number): Point {
+  return {
```

Comment:
nit: this function could call the other scale:

return scale(pt, scale, scale);